core/templates/dev/head/domain/skill/EditableSkillBackendApiService.ts

```diff
@@ -48,6 +48,27 @@ oppia.factory('EditableSkillBackendApiService', [
       });
     };
 
+    var _fetchMultiSkills = function(skillIds, successCallback, errorCallback) {
+      var skills = [];
+      skillIds.forEach(function(skillId) {
+        var skillDataUrl = UrlInterpolationService.interpolateUrl(
```

Comment:
Same here, we are doing multiple get requests for each skill ID.
Instead we should do one single get request and pass in a (comma separated) list of skill IDs in that request.
The response will be a list of skill domains for those skill ids.

Reply:
Done.